guides/micronaut-openapi-generator-server/java/.openapi-generator/FILES

```diff
@@ -0,0 +1,22 @@
+.gitignore
```

Comment:
Should we have this file as part of the complete solution? Or should we delete it?

Reply:
This and the version file is not used by the generator in any way. It seems to be some kind of debug file or something to know what files were generated and by version in case user needs to know.
It can be skipped by user by setting the `-D generateMetadata=false` option.
We can delete it. But if someone will try running the generation themselves, they will have it.